### ISSUE TITLE
Document for Prettier Linter Integration

### ISSUE THREAD

**Issue by xiaoxiangmoe** [prettier/prettier-vscode#1047]
> Prettier Linter Integration (advanced)
>
> This extension will automatically detect when you have these extensions installed and use them instead of prettier by itself. For configuration of these linter integrations, see their respective documentation.

Where is the documentation for it?
I'm using `prettier.stylelintIntegration` and `prettier.tslintIntegration` now.

**Comment by mrm007** [comment]
There's no alternative for `prettier-stylelint`. Stylelint does not recommend using autofixing errors (`stylelint --fix`), [as per docs](https://stylelint.io/user-guide/cli#autofixing-errors):

> Note: It's an experimental feature. It currently does not respect special comments for disabling stylelint within sources (e. g. `/* stylelint-disable */`). Autofixing will be applied regardless of these comments.

Example:

```css
/* stylelint-disable */
:export {
  sizeLarge1X: 11;
  sizeLarge2X: 12;
  sizeRoot: 0;
  sizeSmall1X: 1;
  sizeSmall2X: 2;
}
```

becomes

```css
/* stylelint-disable */
:export {
  sizelarge1x: 11;
  sizelarge2x: 12;
  sizeroot: 0;
  sizesmall1x: 1;
  sizesmall2x: 2;
}
```

**Comment by tsharms** [comment]
I got the same problem, I've followed the prettier [docs](https://prettier.io/docs/en/integrating-with-linters.html) to include Stylelint. But it looks like my own Stylelint config is ignored. My `.stylelintrc` is as follows: 

```
{
    "extends": ["stylelint-rodesk", "stylelint-prettier/recommended"]
}
```

Autofixing Stylelint errors was working in version 2.3.0 with `prettier.stylelintIntegration` enabled. Can someone help out?
